File: cropme/src/main/java/com/takusemba/cropme/ScaleAnimator.java
package com.takusemba.cropme;

/**
 * ScaleAnimator
 **/
interface ScaleAnimator {

    /**
     * scale image
     *
     * @param scale how much image scales
     **/
    void scale(float scale);

    /**
     * rescale image when image is too much big or small
     **/
    void reScaleIfNeeded();
}

File: cropme/src/main/java/com/takusemba/cropme/ScaleAnimator.kt
package com.takusemba.cropme

/**
 * interface to scale Image.
 */
internal interface ScaleAnimator {

  /**
   * scale image
   *
   * @param scale how much image scales
   */
  fun scale(scale: Float)

  /**
   * rescale image when image is too much big or small
   */
  fun reScaleIfNeeded()
}
